src/ext-test/java/org/opentripplanner/ext/legacygraphqlapi/TestHeaders.java

```diff
@@ -0,0 +1,68 @@
+package org.opentripplanner.ext.legacygraphqlapi;
+
+import jakarta.ws.rs.core.Cookie;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.MultivaluedMap;
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+public class TestHeaders implements HttpHeaders {
+
+  private static final RuntimeException NOT_IMPLEMENTED = new IllegalStateException(
+    "Not implemented yet!"
+  );
+
```

Comment:
```suggestion
```
This is an interesting twist ;-) When is the StackStace set? And what happens if you throw this exception in two places at the same time ...